src/data-sources/collection/debank/types.ts

```diff
@@ -0,0 +1,41 @@
+export type NFTRaw = {
```

Comment:
are you sure that every field here is non-nullable? seems to good to be true...

Reply:
https://docs.cloud.debank.com/en/readme/api-pro-reference/user#get-user-nft-list got it from debank specification